### ISSUE TITLE
Failing pa11y test on npm run build

### ISSUE THREAD

**Issue by rossmullen** [govau/design-system-components#1047]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/design-system-components/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

`npm run test` fails on the pa11y automated tests on the `tags` component identifying several failures of colour contrast in the `au-tag--dark` theme variant.

## What I expected to happen

`npm run test` to not encounter any colour contrast failures.

## Reproducing

- Module name: `n/a`
- Module version: `n/a`
- Pancake version: `n/a`
- Node version: `10.19.0`
- npm version: `6.14.4`

Steps to reproduce:

1. `npm i`
2. `npm run bootstrap`

## Attachments

![image](https://user-images.githubusercontent.com/43454021/119440201-33024500-bd67-11eb-87c8-eaa243e58f82.png)
